.github/workflows/api-integration-tests.yml

```
@@ -2,98 +2,64 @@ name: API Integration Tests
-        run: bun run db:setup
-
-      - name: Run linting
-        run: bunx @biomejs/biome check src/
-
+          DATABASE_URL: "postgresql://test:test@localhost:5432/test"
+          IPFS_KEY: "test-key"
+          IPFS_GATEWAY_WRITE: "https://ipfs.io"
+          RPC_ENDPOINT: "https://rpc.testnet.example.com"
+          CHAIN_ID: "80451"
+          DEPLOYER_PK: "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)